<!-- -->
<launch>

	<node name="driverless_node" pkg="driverless" type="driverless_node" output="screen" launch-prefix="gnome-terminal -x"> <!--  -->
		
		<rosparam file="$(find driverless)/config/vehicle_params.yaml" command="load"/>
		<!-- main -->
		<param name="is_offline_debug" value="false"/>
		<param name="max_speed" value="1"/>
		<param name="odom_topic" value="/slam_fused_odom"/>
		
		
		<!-- path_tracking -->
		<param name="min_foresight_distance" value="4"/>
		<param name="max_side_accel" value="1"/>
		<param name="foreSightDis_speedCoefficient" value = "0.3"/>
		<param name="foreSightDis_latErrCoefficient" value = "1.0"/>
		<param name="car_state" value="/car_state"/>

			<!-- PID -->
		<param name="d_omega" value="10.0"/>
		<param name="i_ki" value="0.5"/>
		<param name="p_kp" value="5.0"/>
		<param name="tolerate_laterror" value = "0.05"/>	<!-- 容忍横向偏差 -->
		<param name="control_rate" value="30.0"/>
		<param name="steer_clearance" value="5.0"/>
			<!-- livox -->
		<param name="is_object" value="/min_dis_obj"/>
		<param name="safety_distance" value="4.0"/>
		<param name="timeout" value="0.3"/>
		<!-- reverse control test-->
		<param name="reverse_test" value="false"/>
		<param name="reverse_path_flip" value="false"/>
		<param name="reverse_path_file" value=""/>
	</node>
</launch>
